packages/components/psammead-brand/README.md

```diff
@@ -14,6 +14,7 @@ The `Brand` component provides the BBC service logo (as SVG), nested inside a st
 | --------- | ------ | -------- | ------- | ------------ |
 | brandName | String | yes      | N/A     | `'BBC News'` |
 | svg | Object | yes | N/A | { group: `(<g fillrule="evenodd"><path d="M84.32" /></g>)`, viewbox: { height: 24, width: 167.95 }, ratio: 6.9979 } |
+| svgHeight | Object | yes | N/A | `{ groupA: 16, groupB: 20, groupD: 24 }` |
```

Comment:
Nice API, tbh it would be nice if this wasnt required, perhaps we can add the default of `22, 22, 24` as thats what the majority of services, inc news, use.
 
Makes it a bit easier for people outside of simorgh to use :)